user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+will be able to :ref:`download <user-guide-getting-started-my-user-outlook>` the latest version 
+of OroCRM add-in for Outlook. 
+
+If :ref:`OroCRM is synchronized with Outlook <user-guide-synch-outlook>`, you can use the add-in to synchronize 
+contacts, tasks, and calendar events between OroCRM and your Outlook. You can associate emails to accounts, contacts, 
+opportunities, and cases and create leads, opportunities and cases in OroCRM based on emails from the Outlook side bar.
+
+This is a general capability that can improve the user experience for all enterprise users using both OroCRM and 
+Outlook. 
+
+
+|
+
+.. image:: ./img/intro/user_outlook.png
+
+|
+
+
+.. _admin-capabilities-address-dic:
+
+Read address dictionaries
+-------------------------
+
+The capability enables access to countries, regions, and address types via API. It has to be set to *"System"*, if 
+the Lead creation via Outlook should be supported. This capability should be available to system administrators or 
+integrators, authorized to access OroCRM via API.
+
+
+
+.. _admin-capabilities-search:
+
+Search
+------
+
+The capability enables users to use the :ref:`search <user-guide-getting-started-search>` 
+functionality to find specific records quickly.
+
+This is a general capability that can improve the overall user-experience.
+
+The setting does not influence the user's ability to :ref:`search by tag <user-guide-getting-started-search-tag>`.
+
+
+.. _admin-capabilities-campaign-emails:
+
+Send Campaign Emails
+--------------------
+
+The capability enables users to :ref:`send emails within OroCRM email campaigns <user-guide-email-campaigns-send>`.
+This way, marketing associates can sed a personalized template-based letter to multiple users.
+
+The setting does not effect the user's ability to create and edit :ref:`email campaigns <user-guide-email-campaigns>` 
+and should only be allowed to employees authorized to send email campaigns. These may be emails to potential customers, 
+other system users, etc.  
+
+
+|
+
+.. image:: ./img/roles/email_campaign.png
+
+|
+
+
+.. _admin-capabilities-share-grid:
+
+Share grid view
+---------------
+
+The capability enables users to share the :ref:`grid <user-guide-ui-components-grids>` views that they have configured, 
+this way the user can :ref:`adjust a grid <user-guide-ui-components-grids-adjust>` and share it with other users.
+
+This is particularly useful for team-leads and heads of departments, who can share grids with their subordinates, but 
+may also be available to other users.
+
+|
+
+.. image:: ./img/roles/grid_share.png
+
+|
+
+ 
+.. _admin-capabilities-system-info:
+
+System Information
+------------------
+
+The capability enables users to see view the system information page. This page contains the list of Oro Packages and 
+third-party packages installed  and is usually used by the system administrators and integrators only.
```

Comment:
Edit: This capability lets users view the system information page. This page contains the list of Oro packages and third-party packages that are installed, and is usually only used by system administrators and integrators.